homeassistant/components/bmw_connected_drive.py

```diff
@@ -103,3 +111,21 @@ def update(self, *_):
     def add_update_listener(self, listener):
         """Add a listener for update notifications."""
         self._update_listeners.append(listener)
+
+    def async_add_to_group(self, vehicle, entity_id: str):
```

Comment:
I don't think we should create groups automatically. Let the users do that if the want.

Reply:
OK, removed the groups again.

I though it would be handy, as a vehicle might end up with 10-15 individual sensors. And creating the groups manually is annoying...